src/components/core/AnimeCard.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react'
+import {
+    Image, Text, View, StyleSheet, Dimensions, Pressable
+} from "react-native";
+import { Ionicons } from '@expo/vector-icons';
+import { useAppSelector } from '../../hooks/navigation';
+import { firebase, store } from '../../config/firebase';
+
+const SCREEN_HEIGHT = Dimensions.get('window').height
+const SCREEN_WIDTH = Dimensions.get('window').width
+
+
+const AnimeCard = ({ props }) => {
+
+    //console.log("111", props )
+
+    const userId = useAppSelector(state => state.auth.fireBaseToken);
+    const [like, setLike] = useState(false);
+
+    const handleLike = () => {
+
+        if (!like) {
+            store
+                .collection('likes')
+                .doc(userId)
+                .set(
+                    { [props.item.id]: props.item.title.romaji }, { merge: true }
+                )
```

Comment:
If you're working with firebase, it's usually better to move it to separate service. This way you will be easily able to exchange it to another DB or something. It's called dependency injection.